src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -376,6 +378,21 @@ impl Annotation {
         self.get(CONTAINER_TYPE_KEY)
     }
 
+    /// Get the annotation of cpu quota for sandbox
+    pub fn get_sandbox_cpu_quota(&self) -> Option<String> {
```

Comment:
The type of CPU quota need to be verified here, it need to be `i64`. `get_value` function defined at line 345 could help you to enable that.

Reply:
Fixed